File: android/versioned-abis/expoview-abi40_0_0/src/main/java/abi40_0_0/expo/modules/sqlite/SQLitePackage.java
package abi40_0_0.expo.modules.sqlite;

import android.content.Context;

import java.util.Collections;
import java.util.List;

import abi40_0_0.org.unimodules.core.BasePackage;
import abi40_0_0.org.unimodules.core.ExportedModule;

public class SQLitePackage extends BasePackage {
  @Override
  public List<ExportedModule> createExportedModules(Context context) {
    return Collections.singletonList((ExportedModule) new SQLiteModule(context));
  }
}

File: android/versioned-abis/expoview-abi43_0_0/src/main/java/abi43_0_0/expo/modules/sqlite/SQLitePackage.kt
package abi43_0_0.expo.modules.sqlite

import android.content.Context
import abi43_0_0.expo.modules.core.BasePackage
import abi43_0_0.expo.modules.core.ExportedModule

class SQLitePackage : BasePackage() {
  override fun createExportedModules(context: Context): List<ExportedModule> {
    return listOf(SQLiteModule(context))
  }
}
